docs/iris/src/whatsnew/contributions_1.10/deprecate_2015-Dec-01_pp-custom-save-rules.txt

```diff
@@ -0,0 +1,7 @@
+ * deprecated the custom pp save rules mechanism implemented by the methods
+   :meth:`iris.fileformats.pp.add_save_rules' and
+   :meth:`iris.fileformats.pp.reset_save_rules'.
+   The methods :meth:`iris.fileformats.pp.as_fields',
+   :meth:`iris.fileformats.pp.as_pairs'
+   and :meth:`iris.fileformats.pp.save_fields' provide alternative means of
```

Comment:
This "method"/`:meth:` -> "function"/`:func:` change applies lots of other places too.